events/ecs_container_instance.go

```diff
@@ -36,10 +36,10 @@ type ECSContainerInstanceEventAttribute struct {
 }
 
 type ECSContainerInstanceEventResource struct {
-	Name           string    `json:"name"`
-	Type           string    `json:"type"`
-	IntegerValue   int       `json:"integerValue,omitempty"`
-	StringSetValue []*string `json:"stringSetValue,omitempty"`
```

Comment:
I think the schema says you got this right the first time around :) https://github.com/aws/aws-lambda-go/pull/502#discussion_r1174432104

Reply:
:+1: git reset https://github.com/aws/aws-lambda-go/commit/7316ef154a6ae107e075ea453f5133615290b665